frontend/src/views/Game.vue

```diff
@@ -12,6 +12,15 @@
         :cards-fixed-quantity="5"
         :is-enemy="true"
       />
+      <player-avatar
+        :user-avatar="enemyAvatar"
+        :is-enemy="true"
+        class="avatar__container__enemy"
+        @mouseup="(event) => attackPlayer(event)"
```

Comment:
```suggestion
        @mouseup="attackPlayer"
```

Tu peux juste faire ca, ca va passer le param auto